scripts/mutation-test.sh

```diff
@@ -19,13 +19,13 @@ else
     # Only consider the following:
     # * go files in types, keeper, or module root directories
     # * ignore test and Protobuf files
-    go_file_exclusions="-type f ! -path */client/* -name *.go -and -not -name *_test.go -and -not -name *pb* -and -not -name module.go"
+    go_file_exclusions="-type f ! -path */client/* -name *.go -and -not -name *_test.go -and -not -name *pb* -and -not -name module.go -and -not -name sim_msgs.go -and -not -name codec.go -and -not -name errors.go"
```

Comment:
Note to reviewer: We get rid of the following files for these reasons

sim_msgs:
- the sim by nature is expecting varying inputs for each run. therefore, it logically follows that changing a sim message may or may not alter the output, depending on the input.

codec.go:
- we don't test this file.

errors.go:
- altering errors.go errors don't cause tests to fail, because we call the error by name, so the mutation appears as the expected result for all tests that expect it. This feels fine.